Sming/SmingCore/Network/HttpClient.cpp

```diff
@@ -115,17 +115,47 @@ void HttpClient::freeSslSessionPool() {
 			free(sslSessionIdPool[key]->value);
 		}
 		free(sslSessionIdPool[key]->value);
+		free(sslSessionIdPool[key]);
 	}
 	sslSessionIdPool.clear();
 }
 #endif
 
+
+void HttpClient::freeRequestQueue() {
+	for(int i=0; i< queue.count(); i ++) {
+		debugf("~RquestQueue");
```

Comment:
Remove all `debugf` calls that you have added. We have more than enough debug data.